test/docs/p2021/2021-rec.html

```diff
@@ -87,7 +87,7 @@ <h2>Status of This Document</h2>
       W3C recommends the wide deployment of this specification as a standard for the Web.
     </p>
     <p>
-      A W3C Recommendation is a specification that, after extensive consensus-building, has received the endorsement of the
+      A W3C Recommendation is a specification that, after extensive consensus-building, is endorsed by the
```

Comment:
What are you trying to test here? The paragraph for REC is not the expecting one. There's still "*the* W3C"

Reply:
It doesn't matter in this case. This REC document is only used to fail the check

Reply:
Will fix this mockup text in manual check PR.

Reply:
It's the only p2021 document for Recommendation and it's not even using the right wording. I also note, it's only used once in the whole test suite so I think that's an issue but I'll mark the discussion as resolved assuming the test suite gets refactored soon and I don't want to delay these fixes any longer